README.lhs

```diff
@@ -349,11 +345,10 @@ warpSettings :: App -> Settings
 warpSettings app = setOnException onEx $ defaultSettings
  where
   onEx _req ex =
-    when (defaultShouldDisplayException ex)
-      $ runLoggerLoggingT app
+    when (Network.Wai.Handler.Warp.defaultShouldDisplayException ex)
```

Comment:
Thought this would be a nice thing to write out explicitly so that you can tell this isn't a function that comes from Blammo.

Reply:
Makes sense. Personally, I might cut down the noise by doing an `as Warp`. Would be just as clear, IMO.